Example/AppDelegate.swift

```diff
@@ -13,6 +13,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         MiniApp.configure()
+        MiniAppSDKUtility.unzipMiniApp(fileName: "js-miniapp-sample", miniAppId: "mini-app-testing-appid", versionId: "mini-app-testing-versionid")
```

Comment:
Can we avoid creating a new public class than create a interface in our miniapp sdk to unzip a bundle and store it in our file system 
```
MiniApp.unzipMiniApp(fileName: "js-miniapp-sample", miniAppId: "mini-app-testing-appid", versionId: "mini-app-testing-versionid")
```
in that way we dont have to initialize another class, we can use our existing unzipping process under the hood, just a suggestion. 
In android we need the context of the host app to access the file asset folders of the host app, we already have the context in our sdk, so writting a utility class might need another initialization with the context as it's an independent class.